src/components/Modals/DeleteRelationship/DeleteRelModal.jsx

```diff
@@ -0,0 +1,83 @@
+import { useSetAtom, useAtomValue } from "jotai";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import fetchData from "../../../utils/axios";
+import {
+  currentDBIdAtom,
+  userAtom,
+  showDeleteRelationshipModalAtom,
+  deleteTargetRelationshipAtom,
+} from "../../../atoms/atoms";
+
+import Modal from "../../shared/Modal";
+import Button from "../../shared/Button";
+import Content from "../SharedItems/Content";
+import ContentWrapper from "../SharedItems/ContentWrapper";
+import Message from "../SharedItems/Message";
+import Loading from "../../shared/Loading";
+
+function DeleteRelModal() {
+  const queryClient = useQueryClient();
+
+  const { userId } = useAtomValue(userAtom);
+  const currentDBId = useAtomValue(currentDBIdAtom);
+  const deleteTargetRelationship = useAtomValue(deleteTargetRelationshipAtom);
+
+  const setShowDeleteRelModal = useSetAtom(showDeleteRelationshipModalAtom);
```

Comment:
```suggestion
  const setShowDeleteRelationshipModal = useSetAtom(showDeleteRelationshipModalAtom);
```

유사한 다른 변수명들도 일관되게 수정하면 좋겠습니다.